project-2/Handling_Missing_Data.ipynb

```diff
@@ -0,0 +1,8520 @@
+{
```

Comment:
<p>Avoid using "inplace" to drop/delete/replace data unless it is absolutely necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Humphery_Ufuoma/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
@tobaadesugba i have removed most instances of inplace=True now, instead i make a new dataframe copy, and i really see the improvement, it helps especially in debugging, the only instances of inplace=True are towards the end of the program when i made last changes